docs/configuration/backends/kubernetes.md

```diff
@@ -81,6 +81,20 @@ See also [Kubernetes user guide](/user-guide/kubernetes).
 # Default: <built-in template>
 #
 # filename = "kubernetes.tmpl"
+
+# Enable IngressEndpoint configuration.
+# This will allow Traefik to update the status section of ingress objects, if desired.
+#
+# Optional
+#
+# [kubernetes.ingressEndpoint]
+#
+# One must be configured.
```

Comment:
I think we have to emphasize more that there are two different configuration possibilities. So either: hostname + IP OR the service name. Or do I understand this wrong?

Reply:
I think the explanation is on the next line.

Reply:
Can we change it to `At least one has to be configured`? This + the next sentence describes everything well then.